### DISCUSSION TITLE
Lession 9: Makefile error

### DISCUSSION THREAD

**Discussion by AztecWarlord** [Cyfrin/foundry-full-course-cu#276] [Q&A] [answered]
when I try to deploy Raffle contract to Sepolia using the:
```make deploy ARGS="--network sepolia"```
command I get this error at the end of deployment:
```Error: expected value at line 1 column 1 make: *** [Makefile:43: deploy] Error 1```
Any assistance will be greatly appreciated.
By the way I updated foundry with the latest version and I have copied and pasted the Makefile.

**Comment by AztecWarlord** [comment, marked as answer]
I figured out that I had my ```ETHERSCAN_API_KEY```  and my ```ETHEREUM_RPC_URL``` mixed up!

**Comment by AbhiSharma1210** [comment]
Hi Friend,
I am getting the same error. I copied the  ETHERSCAN_API_KEY from alchemy. (under Sepolia Testing -> API key button on right -> copy 'API key'). Is this correct or an I doing something wrong?

**Reply by AztecWarlord** [reply]
Did you echo your ``` ETHERSCAN_API_KEY``` in the terminal before calling the ```make``` command? @AbhiSharma1210

**Reply by AbhiSharma1210** [reply]
I was able to resolve this. Actually I was using incorrect API key for ETHERSCAN_API_KEY. I used the one from ethescan and everything worked nicely. 
Thank you.
